### PR TITLE
Fix/dtfs2 4548 gef webpack

### PR DESCRIPTION
**Pull request by davehlewis**
- Updates GEF-UI to same build process as portal
- Removes E2E tests from UI's
- Minor changes to Portal build file and config
- Move GEF-UI to core-js syntax instead of modules

### CODE REVIEW

**Comment by ttbarnes** [comment]
"Removes E2E tests from UI's" What do you mean by this please? I think if a UI is changed the e2e tests for that UI should run?

File: gef-ui/server/utils/helpers.js [hunk 1]
```diff
@@ -137,7 +137,7 @@ const mapSummaryList = (data, itemsToShow, preview = false) => {
       summaryItems = [
         ...(href ? [{
           href,
-          /* Clean-Deep removes any properties with Null value from an Object. Therefore if all
+          /* Clean-Deep removes any properties with Null value = require( an Object. Therefore if al)l
```
Comment (ttbarnes) [inline gef-ui/server/utils/helpers.js:140]: looks like a find and replace error

**Review by ttbarnes — COMMENTED** [review]
Looks great - just found a typo and question about e2e test trigger.

**Review by ttbarnes — APPROVED** [review]
Happy for this to get merged with the typo as long as we fix in the next PR. In the interests of getting hot reloading working for quicker dev time.

The e2e change removes the tests from running on build - will still run in PR.

Nice work

**Review by ttbarnes — DISMISSED** [review]

**Review by ttbarnes — DISMISSED** [review]

**Review by IgnatG — APPROVED** [review]

**Pull request merged**